legend-engine-executionPlan-execution-store-relational-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/driver/vendors/snowflake/SnowflakeManager.java

```diff
@@ -0,0 +1,50 @@
+package org.finos.legend.engine.plan.execution.stores.relational.connection.driver.vendors.snowflake;
+
+import org.eclipse.collections.api.factory.Lists;
+import org.eclipse.collections.api.list.MutableList;
+import org.finos.legend.engine.plan.execution.stores.relational.connection.authentication.AuthenticationStrategy;
+import org.finos.legend.engine.plan.execution.stores.relational.connection.driver.DatabaseManager;
+import org.finos.legend.engine.plan.execution.stores.relational.connection.driver.commands.RelationalDatabaseCommands;
+import org.finos.legend.engine.plan.execution.stores.relational.connection.ds.specifications.SnowflakeDataSourceSpecification;
+import org.finos.legend.engine.shared.core.operational.Assert;
+
+import java.util.Properties;
+
+public class SnowflakeManager extends DatabaseManager
+{
+    @Override
+    public MutableList<String> getIds()
+    {
+        return Lists.mutable.with("Snowflake");
+    }
+
+    @Override
+    public String buildURL(String host, int port, String databaseName, Properties extraUserDataSourceProperties, AuthenticationStrategy authenticationStrategy)
+    {
+        Assert.assertTrue(extraUserDataSourceProperties.getProperty(SnowflakeDataSourceSpecification.SNOWFLAKE_ACCOUNT_NAME) != null, () -> SnowflakeDataSourceSpecification.SNOWFLAKE_ACCOUNT_NAME + " is not set");
+        Assert.assertTrue(extraUserDataSourceProperties.getProperty(SnowflakeDataSourceSpecification.SNOWFLAKE_REGION) != null, () -> SnowflakeDataSourceSpecification.SNOWFLAKE_REGION + " is not set");
+        Assert.assertTrue(extraUserDataSourceProperties.getProperty(SnowflakeDataSourceSpecification.SNOWFLAKE_WAREHOUSE_NAME) != null, () -> SnowflakeDataSourceSpecification.SNOWFLAKE_WAREHOUSE_NAME + " is not set");
+        String accountName = extraUserDataSourceProperties.getProperty(SnowflakeDataSourceSpecification.SNOWFLAKE_ACCOUNT_NAME);
+        String region = extraUserDataSourceProperties.getProperty(SnowflakeDataSourceSpecification.SNOWFLAKE_REGION);
+        String warehouse = extraUserDataSourceProperties.getProperty(SnowflakeDataSourceSpecification.SNOWFLAKE_WAREHOUSE_NAME);
+        return "jdbc:snowflake://" + accountName + "." + region + ".aws.snowflakecomputing.com/?warehouse=" + warehouse + "&db=KNOEMA_RENEWABLES_DATA_ATLAS";
```

Comment:
Is the hard coded db name intentional ? We should be using the 'databaseName' param ?

Reply:
Was not intentional :) ! Thanks for pointing out!